### ISSUE TITLE
tradersMultiSigKeyPair must not be null

### ISSUE THREAD

**Issue by amihailovs** [bisq-network/bisq#910]
I have received a pop up asking to report the above bug issue.

**Comment by amihailovs** [comment]
Could someone advise whether or not the above error message is likely to affect my ability to execute the trades? I have closed the application with an open order in the meantime. Open orders just sit there for days without execution. I wonder if this is due to the technical issue or a lack of takers...

**Comment by cbeams** [comment]
@amihailovs I believe @ManfredKarrer has seen this error before, and he may be able to comment on it. In the meantime, I believe this issue should _not_ affect whether your trades get taken. Depending on the token pairs you're trading and your % distance from market price, offers can take quite a while to be taken.

Also, what version of Bisq are you running? Thanks.

**Comment by amihailovs** [comment]
Hi, I trade BTC/GBP and use 1% to 2% distance from the market. Used to be filled within 24 hours in the past, but more recently it took a few days - not ideal when the price goes up.

My Bisq version is V0.5.3

I was offline for a day, now reopened Bisq, and no longer see my offer to Buy on the market. I can see that the trade deposit of 0.03 BTC is still reserved in offers, and the offer shows only under "Portfolio" => "My open offers", but it does not appear under "Market" nor in "Buy BTC".

I would appreciate if someone could look into this.

Thanks.

**Comment by cbeams** [comment]
Just a heads-up @amihailovs that the team is traveling over the next few days. We'll get back as soon as possible, but if there's a delay, this is why.

**Comment by ManfredKarrer** [comment]
@amihailovs Can you send me your log file (Account/Backup there is a button to open the log file). You can send it by email (manfred at bitsquare dot io). 
Can you check in the settings if you have checked the "Show my own offers in offerbook) checkbox?

**Comment by amihailovs** [comment]
@ManfredKarrer I have just sent the email with requested info. please let me know if there is anything else I can do. Thank you!

**Comment by Flugstein** [comment]
I have the same issue here, the popup comes up everytime i start the application and everytime i want to take any kind of offer. When I try to take an offer I get the popup, I click it away and then I get stuck on "Check if offer is available ..." and after a while I get a "timeout: peer has not responded".

I have already made several successful trades and this problem just came up as I checked today. My last trade was on August 12th without problems. I also cleaned up my PC a bit the last few days, so maybe I broke something.

I also sent you an email with my logfile @ManfredKarrer . I have "Show my own offers in offerbook" enabled in the settings.

**Comment by ManfredKarrer** [comment]
Seems that is from an closed dispute. Can you check if you have received the funds from all disputes (funds/transactions).

**Comment by ManfredKarrer** [comment]
@Flugstein:  The payout was done: https://blockchain.info/address/3GYBG9WqmQLQjtPyspZwnsDjzFvaC1NTWf
Do you see the tx? Do you still get the error at startup?

**Comment by Flugstein** [comment]
@ManfredKarrer Yes the funds are in my available balance and I can withdraw them if I want to. The transaction shows up under funds/transactions.

As I said, the error pops up every time I start the application and also every time I want to take an offer, nothing has changed there. The error started to come up after I got the payout transaction.

Edit: Ohh, I only now saw that the arbitrator wrote me on September 9th because the payout transaction didn't work. He closed the ticket 3 more times, and then I got the payout transaction apparently. 
I didn't even get a notification for that, only now that I clicked on the grayed out support ticket I see the new messages.
